### PR TITLE
Project documentation for Pet Feeder

### PR DESCRIPTION
**Pull request by EdwardAdelin**
### Pull Request Overview

This pull request adds/fixes Documentation and Diagram.

Signed-off-by: Finichiu Eduard Adelin eduard.finichiu@stud.fils.upb.ro

### CODE REVIEW

File: docs/project/2024/eduard.finichiu/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Project Name
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:1]: Please name your project.

File: docs/project/2024/eduard.finichiu/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+The format is
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device                                                                                                               | Usage                  | Price                                                                                                                      |
+| -------------------------------------------------------------------------------------------------------------------- | ---------------------- | -------------------------------------------------------------------------------------------------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:88]:
```suggestion
|-|-|-|
```

File: docs/project/2024/eduard.finichiu/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+| ---------------------------------------------------------------------------------------------------------------      |
+| [Servo Motor](http://www.ee.ic.ac.uk/pcheung/teaching/DE1_EE/stores/sg90_datasheet.pdf)                              | The servo motor        | [14 RON](https://www.optimusdigital.ro/en/servomotors/26-sg90-micro-servo-motor.html?search_query=servo+motor&results=196) |
+| ---------------------------------------------------------------------------------------------------------------      |
+| [Water level sensor](https://circuitdigest.com/microcontroller-projects/interfacing-water-level-sensor-with-arduino) | The water level sensor | [2 RON](https://www.optimusdigital.ro/en/others/272-senzor-de-nivel-al-apei.html)                                          |
+
+## Software
+
+| Library                                                    | Description                                       | Usage                                  |
+| ---------------------------------------------------------- | ------------------------------------------------- | -------------------------------------- | ------------------------------------ |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:102]:
```suggestion
|-|-|-|-|
```

File: docs/project/2024/eduard.finichiu/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+| [tokio](https://docs.rs/tokio/latest/tokio/)               | runtime for writing reliable network applications | Used for speed of network applications |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:103]: I don't think you can use tokio with embedded systems.

File: docs/project/2024/eduard.finichiu/index.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+| [pico sdk rust](https://docs.rs/pico-sdk/latest/pico_sdk/) | rust binding and wrappers                         | pico sdk for rust                      |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:104]: This library is not for the Raspberry Pi Pico.

File: docs/project/2024/eduard.finichiu/index.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+| [pwm](https://docs.rs/pwm-pca9685/latest/pwm_pca9685/)     | Pulse-width modulation                            | traits for the RP2040 microcontroller  | Used for controlling the servo motor |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://www.youtube.com/watch?v=vKdQXICO-r0&ab_channel=MrFlashPick)
+2. [link](https://www.youtube.com/watch?v=U7KqqlYaXgY&ab_channel=NicoleZhang)
+3. [link](https://mi-home.ro/products/xiaomi-smart-pet-food-feeder)
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:113]: Please use a title for your links, that title is displayed to the user.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by TeodoraMiu — DISMISSED** [review]

File: docs/project/2024/eduard.finichiu/index.md (hunk 3 above)
Comment (alexandruradovici) [reply docs/project/2024/eduard.finichiu/index.md]: Is this resolved?

File: docs/project/2024/eduard.finichiu/index.md (hunk 2 above)
Comment (alexandruradovici) [reply docs/project/2024/eduard.finichiu/index.md:89]: Is this resolved?

File: docs/project/2024/eduard.finichiu/index.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+| [Servo Motor](http://www.ee.ic.ac.uk/pcheung/teaching/DE1_EE/stores/sg90_datasheet.pdf)                              | The servo motor        | [14 RON](https://www.optimusdigital.ro/en/servomotors/26-sg90-micro-servo-motor.html?search_query=servo+motor&results=196) |
+| ---------------------------------------------------------------------------------------------------------------      |
+| [Water level sensor](https://circuitdigest.com/microcontroller-projects/interfacing-water-level-sensor-with-arduino) | The water level sensor | [2 RON](https://www.optimusdigital.ro/en/others/272-senzor-de-nivel-al-apei.html)                                          |
+
+## Software
+
+| Library                                                          | Description            | Usage                                                                                                                   |
+| ---------------------------------------------------------------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:103]:
```suggestion
|-|-|-|
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/eduard.finichiu/index.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
+# Project Name
+
+Pet Feeder
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:3]:
```suggestion
# Pet Feeder

```

Add a one line description of your project under the title.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/eduard.finichiu/index.md (hunk 2 above)
Comment (alexandruradovici) [reply docs/project/2024/eduard.finichiu/index.md:90]: Not resolved

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/eduard.finichiu/index.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+The format is
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|
```
Comment (alexandruradovici) [inline docs/project/2024/eduard.finichiu/index.md:90]:
```suggestion
|-|-|-|
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please use `npm start` to verify if it looks correct.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please add your email address to the pull request template.

**Comment by EdwardAdelin** [comment]
> Please add your email address to the pull request template.

I have added it now. Thank you!

**Review by alexandruradovici — APPROVED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**